pom.xml

```diff
@@ -267,8 +266,8 @@
 
                         </bottom>
                         <links>
-                            <link>http://docs.oracle.com/javase/8/docs/api/</link>
-                            <link>http://docs.oracle.com/javaee/7/api/</link>
+                            <link>https://docs.oracle.com/javase/8/docs/api/</link>
+                            <link>https://jakarta.ee/specifications/platform/8/apidocs/</link>
```

Comment:
Shouldn't these be "11" and "9"?

Reply:
Since the APIs need to compiled at the Java SE 8 level, I left the Java 8 reference as-is. Although now that I read up on this a bit, it looks like this plugin will automatically determine the appropriate link to use for the JDK depending on the \<source/> level.  I think I'll remove it altogether (after testing).

The previous version of Jakarta EE 8 (and Java EE 8) referenced the previous version of the API in the links.  Thus, this link used to point at Java EE 7.  Again, I just followed suit and referenced the previous version of Jakarta EE 8.  If I change this to "9", then it will just be pointing at itself, which doesn't make sense.  Maybe this one gets dropped as well (more experimentation required).

Reply:
I reviewed the resulting javadoc with and without these links specified in the pom, and the resulting javadoc links _all_ seem to be working.  (Of course, I didn't check every single one, but I did do a visual review and they seem to be just fine.)   I plan to remove them completely since they no longer seem to provide a purpose.